docs/oceanic/index.md

```diff
@@ -2,20 +2,75 @@
+
+<figure markdown>
+  ![Oceanic Control FIRs in the Pacific](./assets/POCA.png) 
+  <figcaption>Oceanic Control FIRs in the Pacific. Not shown are VATMEX's Oceanic FIRs.</figcaption>
+</figure>
+
+The following Divisions are party to the agreement:
+
+- VATNZ (NZZO, NZCM and NTTT)
+- VATPAC (YMMM, YBBB, NFFF, AGGG, ANAU)
+- VATUSA (KZAK)
+- VATJPN (RJTG)
+- VATMEX (MMFO)
+
+### Control Positions
+
+| Sector Name                          | Callsign            | Frequency        | Logon ID     | Division | Notes                       |
+| ------------------------------------ | ------------------- | ---------------- | ------------ | -------- | --------------------------- |
+| KZAK - Oakland Oceanic               | San Francisco Radio | 11.282 (129.400) | ZAK_FSS      | VATUSA   |                             |
+| YBBB - Brisbane Oceanic              | Brisbane Radio      | 17.904 (124.650) | BN-TSN_FSS   | VATPAC   | Also controls AGGG, ANAU    |
+| YMMM - Melbourne Oceanic             | Melbourne Radio     | 17.904 (124.650) | ML-IND_FSS   | VATPAC   |                             |
+| NZZO - Auckland Oceanic              | Auckland Radio      | 8.867 (129.000)  | NZZO_FSS     | VATNZ    | Can be split into -E sector |
+| NZZO-TSN - Auckland Oceanic (Tasman) | Auckland Radio      | 5.567 (129.500)  | NZZO-TSN_FSS | VATNZ    |                             |
+| NZCM - McMurdo Oceanic               | McMurdo Radio       | 9.032 (128.700)  | NZCM_FSS     | VATNZ    |                             |
+| NFFF - Nadi Oceanic                  | Nadi Radio          | 8.846 (123.600)  | NFFF_FSS     | VATPAC   |                             |
+| NTTT - Tahiti Oceanic                | Tahiti Radio        | 9.116 (125.500)  | NTTT_FSS     | VATPAC   |                             |
+| RJTG - Tokyo Oceanic                 | Tokyo Radio         |                  |              | VATJPN   |                             |
+| MMFO - Mexico                        |                     |                  |              |          |                             |
+
+### Currency Requirements
 
-    A new profile is currently being developed to merge all of these datasets into one.
+In order to utilize your Oceanic Endorsement, and by extension, exersize your rights under the Pacific Oceanic Agreement, you must meet the following criteria:
 
-!!! danger "Heads up!"
+- A Participating Organisation must require a permitted controller to log 1 hour on a permitted oceanic position, within the previous 3 months, for them to be considered active, and
+- A permitted controller who does not meet the hour requirement specified above, will be considered inactive; and
+- An inactive permitted controller may become active again, by performing a buddy session with a permitted controller who is approved to do so by that PO; and
```

Comment:
I'm not sure PO is previously mentioned anyway so might be a bit confusing and require expanding on first time round?

Reply:
could change to "participating division" as mentioned in ## Pacific Oceanic Agreement?